packages/ods/src/components/input/tests/rendering/ods-input.e2e.ts

```diff
@@ -0,0 +1,166 @@
+import type { E2EElement, E2EPage } from '@stencil/core/testing';
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('ods-input rendering', () => {
+  let el: E2EElement;
+  let page: E2EPage;
+  let part: E2EElement;
+
+  async function setup(content: string, customStyle?: string): Promise<void> {
+    page = await newE2EPage();
+
+    await page.setContent(content);
+    await page.evaluate(() => document.body.style.setProperty('margin', '0px'));
+
+    if (customStyle) {
+      await page.addStyleTag({ content: customStyle });
+    }
+
+
+    el = await page.find('ods-input');
+    part = await page.find('ods-input >>> [part="input"]');
+    await page.waitForChanges();
+  }
+
+  it('should render the web component', async() => {
+    await setup('<ods-input></ods-input>');
+
+    expect(el.shadowRoot).not.toBeNull();
+  });
+
+  describe('part', () => {
+    it('should render with custom style applied', async() => {
+      await setup('<ods-input></ods-input>', 'ods-input::part(input) { width: 100px }');
+      const partStyle = await part.getComputedStyle();
+      expect(partStyle.getPropertyValue('width')).toBe('100px');
+    });
+  });
+
+  describe('method:clear', () => {
```

Comment:
not sure this is the right place for those
`rendering` should contains the tests about things that change the actual render (ex: setting is-loading does display a spinner, is-masked add a button, ...)
we can add specific directories like `methods` and `events` that will contains dedicated tests